Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM minimum2scp/systemd-stretch
 
+#RUN [ "cross-build-start" ]
```

Comment:
What is this?
I would like to have non spellcheck changes in separate PRs

Reply:
accident, it belongs to the buildsystem PR
Doing both in parallel caused some things to mix up.
Removed here.